lua/lazy/core/config.lua

```diff
@@ -7,8 +7,13 @@ local M = {}
 M.defaults = {
   root = vim.fn.stdpath("data") .. "/lazy", -- directory where plugins will be installed
   defaults = {
+    -- By default, only LazyVim plugins will be lazy-loaded. Your custom plugins will load during startup.
```

Comment:
tbh I don't really like adding lazyvim specific stuff here. Also this doesn't really explain what the setting does.
but that's up to you in the end

Reply:
That's obviously something I copy pasted without updating the docs. Will fix tomorrow